lms/templates/learner_dashboard/program_details_tab_view.underscore

```diff
@@ -59,12 +75,10 @@
                             <span><%- gettext('Restart my subscription') %></span>
                         </div>
                     <% } else { %>
-                        <%- StringUtils.interpolate(gettext(
-                            subscriptionState === 'active'
-                            ? 'Manage my subscription'
-                            : 'Start {trialLength}-Day free trial'
-                        ),
-                        { trialLength } ) %>
+                        <%- StringUtils.interpolate(
+                            gettext('Start {trialLength}-Day free trial'),
```

Comment:
In other places on the page, I see 'Day' is lowercased. To maintain consistency, I'd recommend lowercase-ing here, too.

Reply:
Fixed